deflate.c

```
@@ -972,9 +927,70 @@ int32_t Z_EXPORT PREFIX(deflate)(PREFIX3(stream) *strm, int32_t flush) {
+    if (s->status == FINISH_STATE && strm->avail_in != 0)   {
+        ERR_RETURN(strm, Z_BUF_ERROR);
+    }
+
+    /* Skip headers if we do raw deflate stream */
+    if (s->status == INIT_STATE && s->wrap == 0) {
+        s->status = BUSY_STATE;
+    }
+
+    /* Write headers if needed */
+    if (s->status != BUSY_STATE){
+        if (deflateHeaders(s, strm) == Z_OK) {
```

Cppcheck (2.20.0)

[error] The code contains unhandled character(s) (character code=194). Neither unicode nor extended ascii is supported.

(unhandledChar)